packages/utils/src/integration/integration.ts

```diff
@@ -34,8 +33,9 @@ interface OnetrustData {
   script: string
 }
 
-const loadOnetrustDataSwitzerland = (): Promise<OnetrustData[]> => {
-  return fetch(CMS_CH_API)
+const loadOnetrustDataSwitzerland = (lang: Language = GERMAN, region: Region = 'CH'): Promise<OnetrustData[]> => {
+  const url = getIntegrationUrl(lang, region, IntegrationType.ONETRUST)
+  return fetch(url)
```

Comment:
should we handle error here?

Reply:
Do you see any benefit in catching the error, as we return a Promise here? The consumer of the library needs the possibility to react on the error him/herself, so we would need to throw an error again.
Or what would be your proposal?
@nobilo

Reply:
i agree that consumer should react, however what in case of loadOneTrustBaloiseSwitzerland , we are there resolving this one  in the lib ?

Reply:
Updated the consumable loadOnetrustDataSwitzerland() function to return a promise, to which the consumer can react. What do you think?

Reply:
yes like it very much :)